src/Robo/Commands/Sync/SyncCommand.php

```diff
@@ -134,7 +134,8 @@ public function syncDb() {
       ->arg($local_alias)
       // @see https://github.com/drush-ops/drush/releases/tag/9.2.1
       // @see https://github.com/acquia/blt/issues/2641
-      ->option('--source-dump', sys_get_temp_dir() . '/tmp.sql')
+      ->option('--source-dump', sys_get_temp_dir() . '/tmp.source.sql')
+      ->option('--target-dump', sys_get_temp_dir() . '/tmp.target.sql.gz')
```

Comment:
Odd these have different extensions?

Reply:
It is odd. I initially had it without the `gz` and it failed on `gzip -d /tmp/tmp.target.sql`. Documentation is poor for these options, but I sifted through the issue queue and stumbled upon this PR that was merged into an old drush version - https://github.com/drush-ops/drush/pull/1996/files. It shows the `target-dump` option with the `gz` extension.